src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -199,7 +201,9 @@ export class ApplicationStatus extends React.Component {
             <p>Are you sure you want to start over?</p>
             <ProgressButton
               onButtonClick={() => this.removeForm(formId)}
-              buttonText="Start a new application"
+              buttonText={
+                formConfig.startNewAppMessage || 'Start a new application'
+              }
```

Comment:
It'd be cleaner to have a single definition `formConfig.startNewAppMessage || 'Start a new application'` assigned to a variable instead of repeating the same check.

Reply:
Just updated.